packages/axios-extension/test/auth/uaa.test.ts

```diff
@@ -70,6 +70,7 @@ describe('UAA', () => {
         });
 
         it('returns the access token without refresh token', async () => {
+            jest.setTimeout(10000);
```

Comment:
Is the default value of 5s not enough for this test? Does it fail often?

Reply:
It is very close to 5s so intermittent, but regular enough.

Reply:
Ouch 🙈  
Is there an issue to look at this test?

Reply:
Not sure, I am not familiar with this code. 
@tobiasqueck What do you think?